notebooks/book1/02/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    ax[1].set_title("PDF of Function $y(x)$ = $x^2$")
</pre><p>Rename it to "Analytical p(y), y(x) = x^2"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>